runtime-testsuite/test/org/antlr/v4/test/runtime/typescript/TsNodeRunner.java

```diff
@@ -35,7 +35,7 @@ protected void initRuntime(RunOptions runOptions) throws Exception {
 	}
 
 	private void npmInstallTsNodeAndWebpack() throws Exception {
-		Processor.run(new String[] {NPM_EXEC, "--silent", "install", "-g", "typescript", "ts-node", "webpack", "webpack-cli"}, null);
+		Processor.run(new String[] {NPM_EXEC, "--silent", "--force", "install", "-g", "typescript", "npx", "tsx", "webpack", "webpack-cli"}, null);
```

Comment:
We're not switching to tax. According to their web page, they don't use tsc, and don't perform type checking. 
Tsc compatibility is a requirement.

Reply:
Using tsx is just an attempt to get the build working. As you saw ts-node failed in GH actions and it fails for me locally. I use ts-node for other projects and know it a bit, but it still gives griefs, while tsx works out of the box.

Reply:
webpack doesn't do any type checking either. During development you would need an extra type checker anyway. VS Code has a project wide TS type checker option, which makes this super easy.

Anyway, I cannot change that back or I will not be able to run the ANTLR4 tests locally.

Reply:
ts-code runs tsc that does type checking.